### ISSUE TITLE
VoiceOver incorrectly speaks negative numbers in Stats Charts

### ISSUE THREAD

**Issue by shiki** [woocommerce/woocommerce-ios#1890]
## Describe the bug

When there is a negative currency like “-$115.00", VoiceOver seems to ignore some of the characters. For the case of `-$115.00`, VoiceOver only speaks **“minus fifteen”**. 

This bug was found when testing #1874 but I don't believe it was caused by that.

## To Reproduce

1. Refund some orders on wp-admin so that the WC Admin dashboard shows a negative revenue
2. Launch the app 
3. Enable the new stats
4. Turn VoiceOver on. 
5. Navigate to the chart containing the negative item.

## Videos

Please see [this video](https://drive.google.com/file/d/1PZUxaFJ8wOKJIAvnpuufueKwZbIuzvyA/view?usp=sharing). 

## Expected behavior

VoiceOver should speak the negative value correctly. 

## Isolating the problem (mark completed items with an [x]):
- [ ] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [ ] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/storefront/).
- [ ] I can reproduce this bug consistently using the steps above.

## Mobile Environment
Please include:
- Device: 
- iOS version: 
- WooCommerce iOS version: 

## WordPress Environment

<details>
```
Copy and paste the system status report from **WooCommerce > System Status** in WordPress admin.
```
</details>

**Comment by rachelmcr** [comment]
This VoiceOver label comes from `StoreStatsV4PeriodViewController.chartSummaryString()`. It's odd because based on the video in the original issue, the string is correct but VoiceOver just doesn't read the string correctly, which suggests to me that it was more of a bug in VoiceOver than in our label. That said, I re-tested it with different negative values and couldn't reproduce the original issue at all; VoiceOver reads the correct string (the correct currency symbol and value) every time.

Closing as this seems to be ok now, but please reopen if you can still reproduce this.
